ibc-core/ics04-channel/types/src/acknowledgement.rs

```diff
@@ -139,3 +142,9 @@ impl From<AcknowledgementStatus> for Acknowledgement {
             .expect("token transfer internal error: ack is never supposed to be empty")
     }
 }
+
+impl From<DecodingError> for StatusValue {
+    fn from(e: DecodingError) -> Self {
+        StatusValue::new(e.to_string()).expect("error message must not be empty")
+    }
+}
```

Comment:
Based on the context [here](https://github.com/cosmos/ibc-rs/pull/1325#discussion_r1746098772), this conversion doesn't make sense.

Reply:
[19bd0ce](https://github.com/cosmos/ibc-rs/pull/1325/commits/19bd0cecedbccbab5a863d270319aadcd796f602)